plugins/lime-plugin-pirania/src/components/createVoucherForm.js

```diff
@@ -2,9 +2,11 @@ import { h } from "preact";
 import { useState } from "preact/hooks";
 import I18n from "i18n-js";
 import style from "../style.less";
+import Loading from "../../../../src/components/loading";
```

Comment:
We set up an alias for `components`, so we can simply import like:
`import Loading from "components/loading";

I researched for 5' if there is an automation/linting tool for ensuring imports use aliases but I didn't find anything convincing.